plutus-core/plutus-core/src/PlutusCore/Pretty/Default.hs

```diff
@@ -16,6 +18,14 @@ prettyPlcDef = prettyPlcClassicDef
 displayPlcDef :: (PrettyPlc a, Render str) => a -> str
 displayPlcDef = render . prettyPlcClassicDef
 
+-- | Pretty-print a value in the debug mode using the classic view.
+prettyPlcDebug :: PrettyPlc a => a -> Doc ann
+prettyPlcDebug = prettyPlcClassicDebug
+
+-- | Render a value to 'String' in the debug mode using the classic view.
+displayPlcDebug :: (PrettyPlc a, Render str) => a -> str
+displayPlcDebug = render . prettyPlcClassicDebug
```

Comment:
I thought I needed them, but it turned out I didn't. Still makes sense to have them, I think.